cmd/local.go

```diff
@@ -266,8 +281,14 @@ var localCmd = &cobra.Command{
 func init() {
 	serverLogsCmd.Flags().BoolVarP(&follow, "follow", "f", false, "follow logs output")
 	localCmd.AddCommand(serverLogsCmd)
+
 	serverUpCmd.Flags().BoolVarP(&loginParam, "login", "l", false, "login to the local instance after starting it")
+	serverUpCmd.Flags().StringVarP(&config.DefaultConfig.Project, "create-project", "p", config.DefaultConfig.Project,
```

Comment:
Naive question but do you see this option colliding with any other project specification in the future of this particular command? 

If not, I would just simply call this -p / --project to align with other commands options.

Reply:
Just wanted make it more clear that project is 'created' in local instance.
In all other places it's more like --"use"-project.